### DISCUSSION TITLE
Regarding  GSoC project "Linux command line (sh/Perl/Python) interface to Google groups"

### DISCUSSION THREAD

**Discussion by srs-sudeep** [unicode-org/icu4x#4742] [Q&A]
Hello everyone!
I am Sudeep Ranjan Sahoo, pursuing my undergrad in CS from IIT Bhilai. I’m really passionate about CLI tools and have hacked up a few of my own. I am proficient in JavaScript and multiple JS frameworks, including React and Node. I also have some practice in Python and Golang through some cool course projects. Through a GSoC experience, I am looking to dive into open source and play a part in the development of the technologies I often find myself using everyday. With respect to this project, I use nchat ([https://github.com/d99kris/nchat](https://github.com/d99kris/nchat)) for WhatsApp and Telegram. Could someone please clarify for me if we are trying to build something like this to interface with Google Groups and LGL?

If so, I had a few more questions. Can I post them on this GitHub discussion thread? Or is there a public slack / IRC where I could interact with everyone involved in the project?

Thanks!

**Comment by sffc** [comment]
@rickmcgowan would be your mentor and can share more information and answer and questions. Be sure to submit your proposal by April 2 at 18:00 UTC!
